test/db/analysis/x86_32

```diff
@@ -42,7 +42,7 @@ aap
 afl.
 EOF
 EXPECT=<<EOF
-0x00401e80 fcn.00401e80
+0x00401e80 data.00401e80
```

Comment:
no it is not. i checked this and is correct. the additional code adds the reference, then aap adds the function which is already named (because the flag exists) thus keeping the name.